packages/web-runtime/src/store/user.js

```diff
@@ -34,17 +34,23 @@ const actions = {
     // clear oidc client state
     vueAuthInstance.clearLoginState()
   },
-  async logout({ dispatch }) {
-    const logoutFinalizer = (forceRedirect = false) => {
-      // Remove signed in user
-      dispatch('cleanUpLoginState')
-      dispatch('hideModal')
-      dispatch('loadSettingsValues')
+  async logout(context) {
+    const logoutFinalizer = (isOauth2 = false) => {
+      // Force redirect
+      if (isOauth2) {
+        const logoutUrl = context.getters?.configuration?.auth?.logoutUrl
```

Comment:
Would it make sense to use configuration.server concatenated with "index.php/logout" as fallback?

Reply:
good idea!